dojo/templatetags/get_different_total_script_id.py

```diff
@@ -0,0 +1,9 @@
+from dojo.models import Finding
+
+from django import template
+
+register = template.Library()
+
+@register.filter
+def count_unique_vulns(findings):
```

Comment:
`...unique_vulns` seems confusing, as it is counting the number of distinct script IDs. Maybe call it `count_distinct_script_ids()`.